docs/src/man/pooling.md

```diff
@@ -19,21 +19,21 @@ A companion type, `NullableCategoricalArray`, allows storing missing values in t
 In addition to representing repeated data efficiently, the `CategoricalArray` type allows us to determine efficiently the allowed levels of the variable at any time using the `levels` function (note that levels may or may not be actually used in the data):
 
 ```julia
-levels(cv)
+CategoricalArrays.levels(cv)
```

Comment:
if `levels` isn't exported from CategoricalArrays, then what other option do we have?